astropy/stats/nan_funcs.py

```diff
@@ -0,0 +1,106 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
```

Comment:
We've not been very good in astropy with module docstrings, but I think here it would be good to have one, explaining that this just provides sped-up versions of `nan*` functions.

Also, similarly to `numpy`, maybe call the module `nanfunctions`?

Reply:
Both done, thanks!